flixel/addons/effects/FlxEffectTrail.hx

```diff
@@ -0,0 +1,188 @@
+package flixel.addons.effects;
+
+import flixel.math.FlxMath;
+import flixel.math.FlxPoint;
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+import openfl.display.BitmapData;
+import openfl.geom.ColorTransform;
+import openfl.geom.Matrix;
+import openfl.geom.Point;
+using flixel.util.FlxArrayUtil;
+
+/**
+ * This creates a trail copying the bitmapData many times.
+ * 
+ * @author adrianulima
+ */
+class FlxEffectTrail implements IFlxEffect
```

Comment:
`FlxTrail` use an array of `FlxSprite` and `FlxEffectTrail` use an array of `FlxPoint`. I thought it might be lighter with points.

`FlxEffectTrail` can be in the middle of `FlxEffectSprite#effects` chain, this may have some use cases.